20210325/MINUTES.md

```diff
@@ -0,0 +1,135 @@
+        - do not make a static list in the agenda
+        - add here issues only that need discussion
+        - one can add points in this section during the meeting (if comment/help/review is necessary)
+    - pre- and post-move hook [#1480](https://github.com/sardana-org/sardana/pull/1480)
+        - some issues found - Daniel, Zibi and Teresa will stay in touch to solve them
+    - Creating pseudocounters from pseudocounters 
+        - it works until one restarts Sardana - then because of the order of adding the elements during startup, Sardana is missing them and 
+        can't start at all
+        - problem seem to occur only on real elements - on dummy ones it works
+    - Apply position formatting to the limits [#1530](https://github.com/sardana-org/sardana/pull/1530)
+        - docstring style is messing the sphinx build: change style on enable napoleon extension (if accepted) [#1533](https://github.com/sardana-org/sardana/pull/1533)  
+    - Docstring styles [#1535](https://github.com/sardana-org/sardana/issues/1535)
+        - docstring should be consistent
+        - we keep the discussion open to see if there is any interest in the community to use a different docstring style than the reStructuredText (reST)
+    - Move regscans to scans [#1540](https://github.com/sardana-org/sardana/pull/1540)
+        - do we need backporting?
+            - in DESY they use custom names
+    - Set stop flags for each element in multiple elements movement [#1539](https://github.com/sardana-org/sardana/pull/1539)
+        - at the moment bug fix is merged to develop after manual tests
+        - automatic tests based on PyTest need some more attention and will be merged later not to delay the Release
+    - Execute post-scan also in case of stop, abort and exception [#1538](https://github.com/sardana-org/sardana/pull/1538)
+        - post-scan hooks are performed in case of an exception - it is good
+        - at the moment they are not called in case of stop or abort since no macros can run after a `Ctrl + C`
+        - possible solutions would be allow only the post-scan macro to run after the `Ctrl + C` or
+        use a callable instead. The preferred solution is the first one, but it has to be evaluated
+        - the second one is already used at DESY to overcome the problem
+    - Cannot recreate measurement group with same name [#1528](https://github.com/sardana-org/sardana/issues/1528)
+        - there was an old solution for Tango8 but it was blocked by a bug
+        - Zibi will find it and try to update it to work with new Tango and Sardana3
+    - Add the possibility to perform a continuous scan with a single point [#1501](https://github.com/sardana-org/sardana/issues/1501)
+        - not executing the last acquisition issue:
+            - Pass nr_interv as negative number which would indicate skipping the last acquisition or add another parameter
```

Comment:
or add another parameter -> or break backwards compatibility and change nr_interv to nr_points directly.